### PR TITLE
Updated the changes for broader access setting

### PR DESCRIPTION
**Pull request by aanchalJainHai**
Summary: ADO Powershell module has controls related to reviewing broader group access against set of permissions.
These permissions were identified long and there is a need to revisit again to update the permission. The below are the controls we identified to revisit.

ADO_Release_AuthZ_Restrict_Broader_Group_Access
ADO_Build_AuthZ_Restrict_Broader_Group_Access

[Control_Details_AzSK.ADO_1.20.0.csv](https://github.com/dsr-microsoft/DevOps-Security-Scanners/files/14220583/Control_Details_AzSK.ADO_1.20.0.csv)

## Checklist
- [x] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [x] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [x] The title of the PR clearly describes the intent of the PR.
- [x] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Build.ps1 [hunk 1]
```diff
@@ -1875,7 +1875,6 @@ class Build: ADOSVTBase
         }
 
         return $controlResult;
-
```
Comment (v-dpardeshi) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Build.ps1:1878]: Remove this extra space

File: src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json [hunk 2]
```diff
@@ -296,7 +296,7 @@
       "Contributors": [
         "Manage build queue",
         "Stop builds",
-        "Override build checkIn validation",
+        "Override check-in validation by build",
```
Comment (v-dpardeshi) [inline src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json:299]: Are all these permissions being copied from the UI?

File: src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json [hunk 3]
```diff
@@ -675,8 +667,8 @@
         "Bypass policies when pushing",
         "Edit policies",
         "Force push (rewrite history, delete branches and tags)",
-        "Policy exempt for push",
-        "Remove others' locks"
+        "Remove others' locks",
```
Comment (v-dpardeshi) [inline src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json:670]: Have you validated this setting?

File: src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json (hunk 2 above)
Comment (aanchalJainHai) [reply src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json:299]: yes

File: src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json (hunk 3 above)
Comment (aanchalJainHai) [reply src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json:670]: yes

File: src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json [hunk 4]
```diff
@@ -369,53 +369,45 @@
...
       "Project Collection Valid Users": [
         "Edit release pipeline",
-        "Delete release definition",
         "Edit release stage",
         "Delete release stage",
         "Manage deployments",
         "Administer release permissions",
         "Delete release pipeline",
         "Delete releases",
-        "Delete release stage",
```
[the remark this answers is not in the document]
Comment (aanchalJainHai) [reply src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json:405]: It's copied two times that's why

**Review by saikumarra — APPROVED** [review]

**Review by v-dpardeshi — APPROVED** [review]

**Pull request merged**